sdk/identity/Azure.Identity.sln

```diff
@@ -3,23 +3,33 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.0.31717.71
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Identity", "src\Azure.Identity.csproj", "{8A856C05-8CC7-4A1A-A3DC-DD628ECD3E2A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Identity", "Azure.Identity\src\Azure.Identity.csproj", "{8A856C05-8CC7-4A1A-A3DC-DD628ECD3E2A}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Identity.Tests", "tests\Azure.Identity.Tests.csproj", "{CD0BE1AA-F99E-48E7-9FFD-F1B288E62B4F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Identity.Tests", "Azure.Identity\tests\Azure.Identity.Tests.csproj", "{CD0BE1AA-F99E-48E7-9FFD-F1B288E62B4F}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{8F748B65-7779-4A71-B2EC-9BA8B9526AB0}"
 	ProjectSection(SolutionItems) = preProject
-		CHANGELOG.md = CHANGELOG.md
-		..\ci.yml = ..\ci.yml
-		README.md = README.md
-		..\tests.yml = ..\tests.yml
+		ci.yml = ci.yml
+		tests.yml = tests.yml
```

Comment:
Do we need this and the other SolutionItems stuff?

Reply:
I had them in there for convenience, but I can remove them.